esmero-cantaloupe/Dockerfile

```diff
@@ -145,56 +144,22 @@ RUN curl -OL https://github.com/jasper-software/jasper/archive/version-2.0.32/ja
   && make test \
   && make install
 
-WORKDIR /tmp
-RUN wget $PKGSOURCE && \
-    lzip -d -c GraphicsMagick-$PKGVER.tar.lz | tar -xvf - && \
-    cd GraphicsMagick-$PKGVER && \
-    ./configure \
-      --build=$CBUILD \
-      --host=$CHOST \
-      --prefix=/usr \
-      --sysconfdir=/etc \
-      --enable-magick-compat \
-      --mandir=/usr/share/man \
-      --infodir=/usr/share/info \
-      --localstatedir=/var \
-      --enable-shared \
-      --disable-static \
-      --with-modules \
-      --with-threads \
-      --with-webp=yes \
-      --with-tiff=yes \
-      --with-jpeg=yes \
-      --with-jp2=yes \
-      --with-png=yes \
-      --with-xml=yes \
-      --with-wmf=yes \
-      --with-ttf \
-      --with-gs-font-dir=/usr/share/fonts/Type1 \
-      --with-quantum-depth=$QUANTUMDEPTH && \
-    make && \
-    make install && \
-    cd /tmp && \
-    rm -rf GraphicsMagick-$PKGVER && \
-    rm GraphicsMagick-$PKGVER.tar.lz
-  
-
 # Cantaloupe
 WORKDIR /tmp
 
 RUN apt-get update && apt-get install -y --no-install-recommends maven unzip
 
-RUN curl -OL https://github.com/cantaloupe-project/cantaloupe/archive/refs/heads/release/4.1.zip \
+RUN curl -OL https://github.com/cantaloupe-project/cantaloupe/archive/refs/heads/release/5.0.zip \
  && mkdir -p /usr/local/ \
  && cd /usr/local \
- && unzip /tmp/4.1.zip && cd cantaloupe-release-4.1 && mvn clean package -DskipTests \ 
- && mv target/cantaloupe-4.1.10-SNAPSHOT.zip /usr/local/ \
+ && unzip /tmp/5.0.zip && cd cantaloupe-release-5.0 && mvn clean package -DskipTests \ 
+ && mv target/cantaloupe-5.0.4.zip /usr/local/ \
  && cd /usr/local \
- && unzip cantaloupe-4.1.10-SNAPSHOT.zip \
- && ln -s cantaloupe-4.1.10-SNAPSHOT cantaloupe \
- && rm -rf /tmp/4.1.zip \
- && rm -rf /usr/local/cantaloupe-4.1.10-SNAPSHOT.zip \
- && rm -rf /usr/local/cantaloupe-release-4.1 \
+ && unzip cantaloupe-5.0.4.zip \
+ && ln -s cantaloupe-5.0.4 cantaloupe \
+ && rm -rf /tmp/5.0.zip \
+ && rm -rf /usr/local/cantaloupe-release-5.0.zip \
+ && rm -rf /usr/local/cantaloupe-release-5.0 \
```

Comment:
Could you please at es#diff-42c3f3a30c0a0f08cc306e4954768aa25dc035893aef9281c90c86aa31161e45R169
change the command to 
`RUN adduser -u 100 --home $home $user `
We want to be user the user uses always uid 100 so we do not need to change documentation again

Reply:
```terminal
Step 19/29 : RUN adduser -u 100 --home $home $user
 ---> Running in d0faa2997224
adduser: The UID 100 is already in use.
The command '/bin/sh -c adduser -u 100 --home $home $user' returned a non-zero code: 1
```

Is there maybe some kind of upstream change on UIDs? Checking on a built container without setting the UID shows it's taken by `_apt`: `_apt:x:100:65534::/nonexistent:/usr/sbin/nologin`.